integrations/azure/credentials/write-config-netrc.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2020 Bosch.IO GmbH
```

Comment:
2020 - 2021

and everywhere

Reply:
well the PR wasn't changed in 2021 - but now that I have to change it, I guess it makes sense :D